components/content/Img/Img.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './Img';
+
+export default {
+  component: 'diamond-img',
+  argTypes: {
```

Comment:
Should there be more argTypes here or are they guessed from Img.args?

Reply:
Boolean args are inferred from initial args, aspect ratio and object position are free text so no need to define.